test/functional/feature_asmap.py

```diff
@@ -38,8 +38,12 @@ def expected_messages(filename):
 
 class AsmapTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 1
-        self.extra_args = [["-checkaddrman=1"]]  # Do addrman checks on all operations.
+        self.num_nodes = 2
```

Comment:
Why did you add one more node?

Reply:
It was need for the old approach, but not anymore I guess